### ISSUE TITLE
'feff8l' could not be found 

### ISSUE THREAD

**Issue by killiantallman** [xraypy/xraylarch#262]
I hope someone can help me with this issue; I am not sure what the root of it is. After trying to run a Feff calculation with a given .inp file (same CIF-ish file used for Artemis Feff calculation) I receive an error stating the `**feff8l cannot be found**`. I can successfully read in data from Athena project files and plot data so the software works well to that extent. 

I downloaded the program two different ways: the windows installer and via python. Both give the same error.

I have tried several different ways of entering in the commands and such to perform a Feff calculation. I have Feff calculations already using Artemis. However, from what I can tell, it seems that larch does not take the Feff calculation outputted by Artemis as the former requires a single path per file format. So if there is an easier solution to this issue using Demeter or its previously calculated Feff then that would work as well. 

Thank you in advance, any assistance would be greatly appreciated.

Here are the inputs and outputs I have received:

```
feff = feffrunner(feffinp = '/Users/Killian/Desktop/InputForFeffCalcNi.inp')
feff.run()


Traceback (most recent calls last): 
    feff.run()
   File ReadDataScript.lar, line 1
  File "C:\Users\Killian\xraylarch\lib\site-packages\larch\xafs\feffrunner.py", line 155, in run
    raise Exception("'%s' executable cannot be found" % exe)

Exception: 'feff8l' executable cannot be found
```

and

```
feff6l(folder = '/Users/Killian/Desktop', feffinp ='/Users/Killian/Desktop/InputForFeffCalcNi.inp')


Traceback (most recent calls last): 
    feff6l(folder = '/Users/Killian/Desktop', feffinp ='/Users/Killian/Desktop/InputForFeffCalcNi.inp')
   File ReadDataScript.lar, line 1
  File "C:\Users\Killian\xraylarch\lib\site-packages\larch\closure.py", line 76, in __call__
    return self.func(*args, **kwds)
  File "C:\Users\Killian\xraylarch\lib\site-packages\larch\xafs\feffrunner.py", line 253, in feff6l
    feffrunner.run(exe=exe)
  File "C:\Users\Killian\xraylarch\lib\site-packages\larch\xafs\feffrunner.py", line 129, in run
    self.run(exe=module)
  File "C:\Users\Killian\xraylarch\lib\site-packages\larch\xafs\feffrunner.py", line 155, in run
    raise Exception("'%s' executable cannot be found" % exe)

Exception: 'feff8l_rdinp' executable cannot be found
```

**Comment by newville** [comment]
@killiantallman Thanks and sorry for delay in responding.  I wanted to test this on a Windows10 machine and verify this, but it turns out to be not specific to Windows.

Just to be clear, running `feff8l()` works, as with

    larch> feff8l(folder='fes2', feffinp='feff8.inp')

where `fes2` is a folder with a Feff8 input file name `feff8.inp`.     That's probably the best way to run -- it's the clearest running of `feff8l`.  

But `feffrunner` as you used it should have worked too.   It was broken, but this is now fixed in the master branch.  Thanks!

**Comment by killiantallman** [comment]
Wow! Thank you very very much. After updating it using:

 `C:\Users\Killian\xraylarch\Scripts\conda.exe update -all`

I seem to be getting the correct output feffpath files now after trouble shooting with having the correct .inp file (I actually did not have the correct file previously). I used Artemis to get the correct file because I wasn't sure how to using larch.

I think additions to the online documentation on how to prepare a correct .inp file would be very helpful (the ZnSe.inp on GitHub tipped me off). 

Also, the `feffrunner `commands were successful for me while the `feff8l()` type commands were not. The `feffrunner` created all the feffpath files and associated files but `feff8l()` just created the associated files (seemingly more than `feffrunner`) without the feffpath files. This is not an issue for me as the feffrunner worked perfectly fine.

Thank you again!
